iib/workers/tasks/utils.py

```diff
@@ -634,11 +641,11 @@ def set_registry_auths(registry_auths: Optional[Dict[str, Any]]) -> Generator:
 
 
 @retry(
-    exceptions=IIBError,
-    tries=get_worker_config().iib_total_attempts,
-    delay=5,
-    backoff=2,
-    logger=log,
+    before_sleep=before_sleep_log(log, logging.WARNING),
+    reraise=True,
+    retry=retry_if_exception_type(IIBError),
+    stop=stop_after_attempt(get_worker_config().iib_total_attempts),
+    wait=wait_chain(wait_exponential(multiplier=5)),
```

Comment:
How about using iib_retry_delay here instead of hard coding?

Reply:
I introduced iib_retry_multiplier into config, as delay and jitter have different meanings. Here, I forgot to use the variable, however, I already pushed the repair.